webpack.config.js

```diff
@@ -0,0 +1,39 @@
+const path = require('path');
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+
+module.exports = {
+  mode: 'none',
+  entry: {
+    style: "./styles/index.scss"
+  },
+  output: {
+    path: __dirname,
+  },
+  resolveLoader: {
+    modules: ['node_modules', path.resolve(__dirname, 'loaders')],
+  },
+  module: {
+    rules: [
+      {
+        test: /\.scss$/,
+        use: [
+          MiniCssExtractPlugin.loader,
+          {
+            loader: "css-loader",
+            options: {
+              url: false,
+            },
+          },
+          "convert-zass-functions-loader",
+          "unescape-variables-loader",
+          "sass-loader",
+          "escape-variables-loader"
+        ],
+      },
+    ],
+  },
+  plugins: [
+    new MiniCssExtractPlugin()
+  ],
```

Comment:
To remove the empty `style.js` we could add something like this:

```js
  plugins: [
    new MiniCssExtractPlugin(),
    {
      apply: (compiler) => {
        compiler.hooks.afterEmit.tap('DeleteEmptyJSFile', (compilation) => {
          fs.unlinkSync(`${__dirname}/style.js`);
        });
      }
    }
  ],
```

What do you think?

Reply:
Indeed better than just ignoring it 👍🏼

Done in [d94bfd5](https://github.com/zendesk/copenhagen_theme/pull/305/commits/d94bfd5053dd3934c0ba17ad266580a62d3ce168) - the snippet didn't work so had to go via `compilation.deleteAsset('style.js')`